css/style.css

```diff
@@ -0,0 +1,59 @@
+body {
+    margin: 0px;
+    padding: 0px;
+    min-width: 1200px
+}
+.wrapper {
+    width: 1200px;
+    margin: auto;
+}
+.block {
+    line-height: 150px;
+    height: 150px;
+}
+header .block {
+    width: 400px;
+}
+.block:first-child {
+    float: left;
+}
+header .block:nth-child(2) {
+    float: right;
+}
+.block:last-child {
+    overflow: hidden;
+}
+header, footer, main {
+    width: 100%;
+}
+header {
+    background-color:  #980f00;
+}
+main {
+    height: auto;
+}
+main .wrapper {
+    background: #fddfdf;
+    height: auto;
+}
+.blocks {
+    height: 250px;
+}
+.square {
+    background: yellow;
+    width: 230px;
+    height: 160px;
+    margin: 25px;
+    padding: 10px;
+    float: left;
+}
+.text > p {
+    text-align: center;
+    height: 30px;
+}
+footer {
+    background-color:  chartreuse;
+}
+footer .block {
+    width: 600px;
```

Comment:
what another way do you know for vertical alignment except inline-block and line-height?
let's try it in the footer